res/skins/LateNight/button_hotcue.xml

```diff
@@ -37,8 +37,8 @@
           <ConnectValueFromWidget>false</ConnectValueFromWidget>
         </Connection>
         <Connection>
-          <ConfigKey><Variable name="group"/>,hotcue_<Variable name="number"/>_color_id</ConfigKey>
-          <BindProperty>highlight</BindProperty>
+          <ConfigKey><Variable name="group"/>,hotcue_<Variable name="number"/>_color</ConfigKey>
+          <BindProperty>backgroundColorRgba</BindProperty>
```

Comment:
We do not set the background color here. hotcue_nn_color sets the highlight color.